docs/index.bs

```diff
@@ -3112,7 +3112,19 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. If |request|'s [=request/destination=] is either "<code>embed</code>" or "<code>object</code>", then:
           1. Return null.
       1. Else if |registration|'s <a>active worker</a>'s [=service worker/list of router rules=] is [=list/is not empty=]:
-          1. If running [=Get Router Source=] algorithm with |registration|'s <a>active worker</a> and |request| returns "network", return null.
+          1. Let |source| be a result of running [=Get Router Source=] algorithm with |registration|'s <a>active worker</a> and |request|.
+          1. If |source| is {{RouterSource/"network"}}, return null.
+          1. Else if |source| is {{RouterSource/"cache"}}, then:
+              1. Let |requestResponses| be the result of running [=Query Cache=] with |request|.
+              1. If |requestResponses| is an empty [=list=], return null.
+              1. Else:
+                  1. Let |requestResponse| be the first element of |requestResponses|.
+                  1. Let |response| be a copy of |requestResponse|'s response.
```

Comment:
Is "a copy of" necessary? It doesn't seem well-defined. But I see it is used elsewhere, so maybe it is fine.

Reply:
I was inspired by "Add a copy of requestResponse’s response to responses." in https://w3c.github.io/ServiceWorker/#cache-matchall.  However, I guess it is fine to omit "a copy of".  I have removed that.